benchmarks/run-all.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+source common.sh
+
+echo "SIMPLE COPY:"
+pushd simple_copy > /dev/null
+./run.sh
```

Comment:
nit: could pass arguments given to `run-all.sh` to the subscripts (assuming the arguments are common for all the runs)

Reply:
I think I can make all args to be the same (or similar), it will improve usability not just for users, but also to make the benchmark more flexible, I agree. Let me try something...

Reply:
Honestly, bash's `getopts` is really bad. It assumes that you know all arguments and that it's impossible to do partial parsing (only look at some of the args, not all) and it errors out on any unknown options, even if I chose to handle the error, I can't just continue. So, I'll let it be as is for now and honestly, we should have a more robust driver soon.